lib/formats/xml.flow

```diff
@@ -188,7 +191,7 @@ makeTextParser(parameters : [XmlParsingOption]) -> (string, DynamicBehaviour<Xml
 		\s : string, events : DynamicBehaviour<XmlEvent> -> {
 			// Get rid of all CRs
 			s1 = strReplace(s, "\x0d", "");
-			next(events, XmlTextEvent(unescape(s1)));
+			next(events, XmlTextEvent(if (isUrlParameter753True()) s1 else unescape(s1)));
```

Comment:
The performance impact must be well measured because as I remember, the escaping using XML parsers was really slow - it tooks 15% of CPU in rendering wigi documents. measure it using large documents (10k size at least)